pkg/datapath/linux/ipsec/ipsec_linux_test.go

```diff
@@ -39,11 +39,14 @@ func setupIPSecSuitePrivileged(tb testing.TB) *slog.Logger {
 }
 
 var (
-	path           = "ipsec_keys_test"
-	keysDat        = []byte("1 hmac(sha256) 0123456789abcdef0123456789abcdef cbc(aes) 0123456789abcdef0123456789abcdef\n1 hmac(sha256) 0123456789abcdef0123456789abcdef cbc(aes) 0123456789abcdef0123456789abcdef\n1 digest_null \"\" cipher_null \"\"\n")
```

Comment:
Doesn't changing this test case to have a single line also reduces the scope of existing tests?

Reply:
Here now I'm leaving the variables as they are, just changing the SPI in each of them to make sure the existing tests don't fail because of this reason. In addition, I'm introducing `keysSameSpiDat` which defines two valid keys but with the same SPI: this is used to cover the new case introduced.